packages/syft/tests/syft/lib/torchvision/allowlist_test.py

```diff
@@ -46,10 +49,9 @@ def version_supported(support_dict: Union[str, Dict[str, str]]) -> bool:
 
 @pytest.mark.slow
 def test_allowlist(
-    root_client: sy.VirtualMachineClient, tens: torch.Tensor, pil_img: PIL.Image.Image
+    root_client: sy.VirtualMachineClient, tens: torch.Tensor, pil_img: Any
```

Comment:
Q: Why not PIL.Image.Image?

Reply:
Basically, due to the importing based on pytest.importorskip, the module is considered a variable and mypy does not allow you do typing using variables, for example this is invalid:

```
if rand():
    type = A
else:
   type = B

def my_func() -> type: ...
```